streamwebs_frontend/streamwebs/static/streamwebs/js/sites.js

```diff
@@ -82,29 +75,39 @@ function initialize() {
 
     var latSum = 0, lngSum = 0;
 
-    for (let site of sites) {
-        site.marker = new google.maps.Marker({
+    var markerList = [];
+    var infoWindows = [];
+
+    for (var i = 0; i < sites.length; i++) {
+        var site = sites[i];
+
+        markerList[i] = new google.maps.Marker({
             map: map,
-            position: new google.maps.LatLng(site.lat, site.lng),
+            position: new google.maps.LatLng(site.lng, site.lat),
             title: site.name,
             icon: markers[site.type],
         });
-        site.infoWindow = new google.maps.InfoWindow({
+
+        markerList[i].index = i;
+
+        infoWindows[i] = new google.maps.InfoWindow({
             content: '<p><a href="javascript:submit_map(\'' + site.slug + '\')">' + site.name + '</a></p><p>' + site.description + '</p>',
         });
-        site.marker.addListener('click', () => {
-            for (let otherSite of sites) {
-                otherSite.infoWindow.close();
+
+        markerList[i].addListener('click', function () {
+            for (let otherWindow of infoWindows) {
+                otherWindow.close();
             }
-            site.infoWindow.open(map, site.marker);
+
+            infoWindows[this.index].open(map, markerList[this.index]);
         });
 
-        latSum += site.lat;
-        lngSum += site.lng;
+        lngSum += site.lat;
```

Comment:
I fixed the ``sites`` data migration script, so the coordinates in the db should be correct now. In other words, you don't need to swap them to display map markers. :)